python_modules/dagster/dagster/_core/storage/alembic/versions/039_d3a4c9e87af3_add_asset_policy_evaluations_table.py

```diff
@@ -0,0 +1,42 @@
+"""Add asset policy evaluations table
+
+Revision ID: d3a4c9e87af3
+Revises: 701913684cb4
+Create Date: 2023-05-09 11:50:38.931820
+
+"""
+
+import sqlalchemy as db
+from alembic import op
+from dagster._core.storage.migration.utils import has_table
+from dagster._core.storage.sql import get_current_timestamp
+from sqlalchemy.dialects import sqlite
+
+# revision identifiers, used by Alembic.
+revision = "d3a4c9e87af3"
+down_revision = "701913684cb4"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    if not has_table("asset_policy_evaulations"):
+        op.create_table(
+            "asset_policy_evaulations",
+            db.Column(
+                "id",
+                db.BigInteger().with_variant(sqlite.INTEGER(), "sqlite"),
+                primary_key=True,
+                autoincrement=True,
+            ),
+            db.Column("evaluation_id", db.Integer),
+            db.Column("asset_key", db.Text, index=True),
+            db.Column("evaluation_body", db.Text),
```

Comment:
Having the name of this column match the type of the Python objected that's serialized there would make it easier to find one from the other.